scripts/test/unit

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+set -eu -o pipefail
+
+go test -tags daemon -v $@
```

Comment:
Why aren't these just in the Makefile? These kinds of systems are extremely hard to maintain.